docs/management_api.md

```diff
@@ -34,6 +34,20 @@ This API follows the [ManagementAPIsService.RegisterModel](../frontend/server/sr
 ```bash
 curl -X POST  "http://localhost:8081/models?url=https://torchserve.pytorch.org/mar_files/squeezenet1_1.mar"
 
+{
+  "status": "Model \"squeezenet_v1.1\" Version: 1.0 registered with 0 initial workers. Use scale workers API to add workers for the model."
+}
+```
+Your model might be encrypted via [S3 SSE-KMS](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingKMSEncryption.html). In this case, you need specify the following [environment variables](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html). 
+* AWS_ACCESS_KEY_ID
+* AWS_SECRET_ACCESS_KEY
+* AWS_DEFAULT_REGION
+On EC2 instance, you can create an IAM Role (AWSS3ReadOnlyAccess) for the instance, and then run ts_scripts/get_aws_credential.sh to export AWS_ACCESS_KEY_ID and WS_SECRET_ACCESS_KEY.
+
+And set "s3_sse_kms=true" in HTTP request. For example:
+```bash
+curl -X POST  "http://localhost:8081/models?url=https://torchserve.pytorch.org/mar_files/squeezenet1_1.mar&s3_sse_kms=true"
```

Comment:
After trying this I got a null pointer exception curl -X POST  "http://localhost:8081/models?url=https://torchserve.pytorch.org/mar_files/squeezenet1_1.mar&s3_sse_kms=true"

I fetched your pr and installed dependencies and project successfully on an AWS server. Your instructions seem to imply that squeezenet is already encrypted. This happens if I don't set any of the environment configs, so maybe an error here makes more sense

Reply:
This curl command is just an example. In your test, you need encrypt your model via S3 SSE KMS and run test on it.

Reply:
I see it may be useful to add a few lines on how to encrypt a model with S3 SSE KMS, I think that's an interesting separate scenario. If it's too long maybe in a separate example